modules/core/shared/src/main/scala/gem/enum/GsaoiReadMode.scala

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem
+package enum
+
+import cats.instances.string._
+import cats.syntax.eq._
+import gem.util.Enumerated
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Enumerated type for GSAOI Read Mode.
+ * @group Enumerations (Generated)
+ */
+sealed abstract class GsaoiReadMode(
+  val tag: String,
+  val shortName: String,
+  val longName: String,
+  val ndr: Int,
+  val readNoise: Int,
+  val minimumExposureTime: FiniteDuration,
+  val overhead: FiniteDuration
+) extends Product with Serializable
+
+object GsaoiReadMode {
+
+  /** @group Constructors */ case object Bright extends GsaoiReadMode("Bright", "Bright", "Bright Objects", 2, 28, new FiniteDuration(5, java.util.concurrent.TimeUnit.SECONDS), new FiniteDuration(10, java.util.concurrent.TimeUnit.SECONDS))
```

Comment:
Here again the `minExposureTime` before was `5.3` so you'd want `FiniteDuration(5300, MILLISECONDS)`. It's fine that they appear as milliseconds here in the `case object`, we just want the fractional second value in the database so that it makes sense to the user doing a query.